src/mscorlib/src/System/Runtime/Intrinsics/Vector256.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    [StructLayout(LayoutKind.Sequential, Size = 32)]
+    public struct Vector256<T> where T : struct 
+    {
+        public static bool IsSupported {get;}
+    }
+}
```

Comment:
Nit: Add new lines at the end of the file

Reply:
Done.